comm/src/test/scala/coop/rchain/comm/rp/ConnectSpec.scala

```diff
@@ -15,7 +15,9 @@ import org.scalatest._
 
 import scala.concurrent.duration.{FiniteDuration, MILLISECONDS}
 
-class ConnectSpec extends FunSpec with Matchers with BeforeAndAfterEach with AppendedClues {
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+class ConnectSpec extends AnyFunSpec with Matchers with BeforeAndAfterEach with AppendedClues {
```

Comment:
Please optimize and sort imports with empty line before class definition.

Reply:
[Done](https://github.com/rchain/rchain/pull/3721/commits/580a39185ca80361b7c8fb6f6a7435d90c1536a8).